region/src/main/java/org/motechproject/nms/region/service/impl/LanguageServiceImpl.java

```diff
@@ -57,7 +57,7 @@ public String getSqlQuery() {
                         "from nms_languages l " +
                         "join nms_districts d on  d.language_id_oid = l.id " +
                         "join nms_states s on d.state_id_oid = s.id " +
-                        "join FLW_MODULE_JOIN_CIRCLES_STATES cxs on s.id = cxs.state_id and cxs.circle_id = ?";
+                        "join nms_states_join_circles sxc on s.id = sxc.state_id and sxc.circle_id = ?";
 
```

Comment:
The test container failures should be resolved now. Now, we are back to just a single test failure which I didn't get to debug before the game :)